composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "phpseclib/phpseclib",
+    "name": "phpseclib/phpseclib3",
```

Comment:
should the package name be updated?  I'm for the rest, but this is beyond a code change and has deeper effects. 
If the package name is to change, then maybe phpseclib3 needs to migrate to a new repo?

Reply:
Oh - good catch. I did a `sed` on the codebase and I guess that got caught up in it lol.

I'm not currently in a place where I can change it (I'm traveling and saw the Pyramids today!) but I'll update this as soon as I'm able.

Thanks!